changelog/5521.bugfix.rst

```diff
@@ -0,0 +1 @@
+Responses used in ResponseSelector now support new lines.
```

Comment:
As far as I understand, this would work if we explicitly use `\n` in the response text, right?